src/test/regress/output/multi_insert_select_behavioral_analytics_create_table.source

```diff
@@ -62,3 +62,370 @@ CREATE INDEX is_index3 ON users_table(value_1);
 CREATE INDEX is_index4 ON events_table(event_type);
 CREATE INDEX is_index5 ON users_table(value_2);
 CREATE INDEX is_index6 ON events_table(value_2);
+-- create a helper function to create types/functions on each node
+CREATE FUNCTION run_command_on_master_and_workers(p_sql text)
```

Comment:
> We already use it in two different places. Why create a technical debt while we can fix it easily now?

I don't think that it fixes anything. If I were to use the same function here, I'd have created an order dependency between that test and this test. The proper fix is to add this function to `citustools`, which I don't want to do on this PR.

> Also having different names for the same functionality is not right.

That was intentional. I gave a separate name to prevent error in case we run the tests concurrently.